src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/Address.kt

```diff
@@ -1,5 +1,15 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
 
 data class Address(
+  val country: String?,
+  val county: String?,
+  val endDate: String?,
+  val locality: String?,
+  val name: String?,
+  val number: String?,
   val postcode: String?,
+  val startDate: String?,
+  val street: String?,
+  val town: String?,
+  val type: String?,
```

Comment:
For ease of reading by us and the consumer, and I know I'm being pedantic, but can this be ordered to reflect the structure of an address?
On top of this can we attempt to use the following field definitions:
flat
premise
name
streetAddress
addressLocality
addressTown
addressRegion
addressCountry
postalCode
startDate
endDate
additionalType

Reply:
Actually alphabetical makes more sense for development and machine readings users.